### PR TITLE
Dialog Customization

### PR DESCRIPTION
**Pull request by emme1009**
Hi, 

I updated the dialog to a slightly more modern and fresher look.
Colors and styles can simply be overwritten by copying the colors.xml and styles.xml files to your own project and change them to your needs. 

What can be changed:
- all colors can be customized
- paddings
- texts can be translated
- cornerradius of button and dialog

Few more minor fixes included
- fixed packagename of the example to run it
- upgraded gradle wrapper and kotlin version
- added styles to the card input dialog xml file in order to use colors and styles
- extracted string resources in order to overwrite them with your own language etc.

If you need extra text in the dialog you could simply copy the layout.xml file to your own project and add whatever you need.

I dont know how to style it on iOS so this is Android only. But since I was not able to style it the way I wanted I came up with this fix. Hopefully you can merge it. I would badly need it for my own project.

Screenshot_1591790193
Screenshot_1591790058

### CODE REVIEW

**Comment by Zireael26** [comment]
@emme1009 you can use it from your own fork and branch. In fact, I am using it from your fork and the branch feat/customization

Add this to the pubspec.yaml file of the project you want to use it in
`  stripe_payment:
    git:
      url: git://github.com/emme1009/flutter_stripe_payment.git
      ref: feat/customization`

**Comment by emme1009** [comment]
@Zireael26 Cool I never knew that this would work. Thank you.

**Comment by Zireael26** [comment]
Yeah man! You can get any package directly from a git repo. I have been doing this with Hive because the one on pub has issues with Flutter test dependency.

File: example/android/app/build.gradle [hunk 1]
```diff
@@ -28,8 +28,8 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "de.jonasbark.stripepaymentexample"
-        minSdkVersion 19
+        applicationId "jonasbark.de.stripepaymentexample"
```
Comment (jonasbark) [inline example/android/app/build.gradle:31]: please revert this line - de.jonasbark is the correct format

File: example/android/app/build.gradle [hunk 2]
```diff
@@ -28,8 +28,8 @@ android {
...
+        minSdkVersion 21
```
Comment (jonasbark) [inline example/android/app/build.gradle:32]: this would be a breaking change I'm afraid

**Comment by jonasbark** [comment]
It certainly looks better but there's a catch
1. internationalization should be controllable within dart, e.g. pass the strings as options
2. this PR would only make it work on iOS

**Comment by emmaxCreative** [comment]
@jonasbark 
1. I have no clue how to do such a thing. Never made a package or somethig similar. But at least we can provide some different languages instead of only an english hardcoded string and we can overwrite them in our own projects if we need more languages and or a different buttontext. Which is not possible if the buttontext is hardcoded in the xml.

2. You mean only Android ? I don´t have a Mac so I cannot develop or test changes for iOS.

File: example/android/app/build.gradle (hunk 2 above)
Comment (emmaxCreative) [reply example/android/app/build.gradle]: I revert it back. It is also runnable with 19

File: example/android/app/build.gradle (hunk 1 above)
Comment (emmaxCreative) [reply example/android/app/build.gradle]: Sure I will revert it. But you have different package names throughout the example and it was only runnable with this packagename and not the provided one. 

Thats why I had to change the packagename to run in:
image

I have now fixed the packagename in the MainActivity. Now its working with your package name.

Comment (emme1009) [reply example/android/app/build.gradle]: Sure I will revert it. But you have different package names throughout the example and it was only runnable with this packagename and not the provided one.

Thats why I had to change the packagename to run in:
image

I have now fixed the packagename in the MainActivity. Now its working with your package name.

File: example/android/app/build.gradle (hunk 2 above)
Comment (emme1009) [reply example/android/app/build.gradle]: I revert it back. It is also runnable with 19

**Comment by emme1009** [comment]
@jonasbark 
1. I have no clue how to do such a thing. Never made a package or somethig similar. But at least we can provide some different languages instead of only an english hardcoded string and we can overwrite them in our own projects if we need more languages and or a different buttontext. Which is not possible if the buttontext is hardcoded in the xml.

2. You mean only Android ? I don´t have a Mac so I cannot develop or test changes for iOS.

**Pull request closed without merging**